src/store/fenceTypes.ts

```diff
@@ -1,9 +1,14 @@
-export type FenceName = string;
-
 export type AclsByFence = {
   [fenceName: string]: string[];
 };
 
 export type UserAcls = string[];
 
 export type FencesAllConcatenatedAcls = string[];
+
+export enum FenceName {
+  gen3 = 'gen3',
+  dcf = 'dcf',
```

Comment:
doesn't that enum showing up a couple of places?

Reply:
(see chat for more details): before we had two separate constants `DCF` and `GEN3` that were used in many places in the app. I centralized them into an enum. Some files had many `DCF` and `GEN3`  floating around and I didn't want to bother relplacing each and everyone of them with `myEnum.gen3`, for instance. So for these files, at their top I created a local constant like `const DCF = myEnum.dcf` so that the code had very little change to it. It's a bit lazy but I had so many other things to do in that pr that I consider it a minor issue.